### ISSUE TITLE
[ Mohist Server] Compatibility Support Please ! 1.20.1 , Forge 47.3.11

### ISSUE THREAD

**Issue by conetopia** [MehVahdJukaar/Supplementaries#1193]
Decent amount of people using Mohist Servers these days.
Hoping to reach out, and see if some compatibility can be added in the mod, so its able to run on Mohist Server.

**Comment by MehVahdJukaar** [comment]
no. This is a mod built fo 2 mod loades, fabric and forge. Mohist would be a third which would require a third version of the mod there. Its general consensus in the modding community that hybrid servers should not be used and are not supported. Use mods! there's nothting they cant do, why do you need 2 competing badly patched togethermod loaders in same game? its just asking for issues, likely this one too its a mohist one. That thingwill inevitably have issues so dont use it.

**Comment by conetopia** [comment]
@MehVahdJukaar  Needed Mohist server ( not really even for plugins ) but its so i can run it on 
The Multicraft Platform,  as it requires a .jar , and forge doesn't have a server.jar for MC 1.19.x & 1.20.1 .

Hopefully your still around in 2-3 years from now, and I would be better off waiting for maybe a minecraft 1.20.6 or newer version of this mod, as they have forge server.jars again.

**Comment by MehVahdJukaar** [comment]
Forge can't have a jar as redistributing the entire game in a jar is against the terms of service for obvious reasons. Ask around on forge, maybe things changed recently
